Original Source: http://git.savannah.gnu.org/cgit/guix.git/tree/gnu/packages/patches/torsocks-dns-test.patch


Skip DNS tests that rely on the ability to look up arbitary
host names.

--- torsocks/tests/test_dns.c	2015-11-10 18:30:53.955941984 +0100
+++ torsocks/tests/test_dns.c	2015-11-10 18:31:02.199941892 +0100
@@ -134,11 +134,8 @@ static void test_getaddrinfo(const struc
 int main(int argc, char **argv)
 {
 	/* Libtap call for the number of tests planned. */
-	plan_tests(NUM_TESTS);
+	plan_tests(1);

-	test_getaddrinfo(&tor_check);
-    test_gethostbyname(&tor_dir_auth1);
-	test_gethostbyaddr(&tor_dir_auth2);
 	test_getaddrinfo(&tor_localhost);

     return 0;
